website/content/docs/ecs/compatibility.mdx

```diff
@@ -0,0 +1,24 @@
+---
+layout: docs
+page_title: Compatibility Matrix
+description: Compatibility Matrix for Consul ECS
+---
+
+# Compatibility Matrix for Consul on ECS
+
+For every release of Consul on ECS, `consul-ecs` binary and `consul-ecs` terraform module is updated. The terraform module and binary are tightly coupled. Ex: consul-ecs binary 0.4.1 must use consul-ecs terraform 0.4.1.
+
+## Supported Consul versions
+
+### Version 0.2.0 and above
+
+
+| Consul Version | Compatible consul-ecs Versions  |
+| -------------- | ------------------------------- |
+| 1.10.x         | 0.2.x                           |
+| 1.11.x         | 0.3.0, 0.4.x                    |
```

Comment:
Suggest descending order for the versions; i.e., that we put the most recent versions of Consul/Consul on ECS at the top. I think this has a more natural feel and is consistent with how it's presented on the k8s compat matrix.

```suggestion
| 1.12.x         | Not available yet               |
| 1.11.x         | 0.3.0, 0.4.x                    |
| 1.10.x         | 0.2.x                           |
```